api/client/commands.go

```diff
@@ -2750,7 +2761,7 @@ func (cli *DockerCli) CmdStats(args ...string) error {
 	if len(errs) > 0 {
 		return fmt.Errorf("%s", strings.Join(errs, ", "))
 	}
-	for _ = range time.Tick(500 * time.Millisecond) {
+	for range time.Tick(500 * time.Millisecond) {
```

Comment:
@duglin can we make sure we keep compatibility with Go 1.3 ? Jenkins should have failed :(